doc/api/api_changes/2017-12-20-TH.rst

```diff
@@ -0,0 +1,5 @@
+Deprecations
+````````````
+
+`.Legend.draggable()` is drepecated in favor of `.Legend.set_draggable()`.
```

Comment:
Do these links work w/ ()?

Reply:
Yes. Sphinx can handle this. I find it convenient to use brackets in method links, because it's more obvious in the source code.